extension/popup.html

```diff
@@ -20,40 +20,6 @@
       <select id='station' name='station'>
         <option value=''>Choose a station</option>
         <option value='bagel'>BAGeL Radio</option>
```

Comment:
I think you want to remove BAGeL Radio, too, since the others go away. Looks like it did get deleted in the .haml file.